.claude/skills/maintain-pr/scripts/fetch-review-threads.sh

```
@@ -0,0 +1,78 @@
+                  path
+                  line
+                  startLine
+                  author { login }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  ')
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)